forge-gui/res/cardsfolder/upcoming/bank_job.txt

```diff
@@ -0,0 +1,13 @@
+Name:Bank Job
+ManaCost:1 B R
+Types:Enchantment
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigExile | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, exile the bottom creature card of your library. You may cast that card this turn. At the beginning of the next end step, if that card is still exiled, put it into your graveyard and create a Treasure token.
+SVar:TrigExile:DB$ ChangeZoneAll | ChangeType$ Card.YouOwn+BottomLibrary_Creature | Origin$ Library | Destination$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ MayCast | Duration$ Permanent | RememberObjects$ Remembered | ExileOnMoved$ Exile | Triggers$ EndStep | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:MayCast:Mode$ Continuous | Affected$ Card.IsRemembered+nonLand | MayPlay$ True | EffectZone$ Command | AffectedZone$ Exile | Description$ You may cast that card this turn.
```

Comment:
Another problem: if you end the turn with _Obeka_ the delayed trigger will happen next one, but you should already no longer be able to cast it :/

Reply:
To fix this, I split it into an effect for the static and a delayedtrigger

Reply:
Think I have this solved now